packages/calcite-components/src/components/popover/popover.e2e.ts

```diff
@@ -699,4 +700,64 @@ describe("calcite-popover", () => {
       });
     });
   });
+
+  describe("theme", () => {
+    const popoverHTML = html`
+      <calcite-popover heading="I'm a heading in the header using the 'heading' prop!" closable>
+        Lorem Ipsum
+      </calcite-popover>
+    `;
+
+    describe("default", () => {
+      themed(popoverHTML, {
+        "--calcite-popover-background-color": {
+          shadowSelector: `.${CSS.calciteFloatingUIAnim}`,
+          targetProp: "backgroundColor",
+        },
+        "--calcite-popover-border-color": [
+          {
+            shadowSelector: `.${CSS.calciteFloatingUIAnim}`,
+            targetProp: "borderColor",
+          },
+          // { shadowSelector: ".arrow::before", targetProp: "outline" },
```

Comment:
Having hard time targeting this. It's a class on FloatingArrow component within Popover.

Reply:
`.arrow` should be `.calcite-floating-ui-arrow`. It should've gotten renamed in https://github.com/Esri/calcite-design-system/pull/6847/files#diff-2c88956271241eaf8204f4d12134f4fff58f10aeac5dc0dd8e18ad9e21f37607.

@driskull Should the `::before` selector be moved into `_floating-ui.scss` or should it stay in `popover.scss`?

Reply:
Yeah probably. It seems like this class is only on tooltip and popover and they are the only components that render an arrow. So it probably needs to move.